apidoc/Altimeter.yml

```diff
@@ -0,0 +1,50 @@
+---
+name: Modules.CoreMotion.Altimeter
+platforms: [iphone, ipad]
+summary: | 
+    Allows Titanium client applications to access CoreMotion's Altimeter APIs. 
+    Note: This API is only available in iOS 8 and later.
+since: "8.3.0"
+extends: Titanium.Proxy
+createable: true
+
+methods:
+  - name: isSupported
```

Comment:
I think this API is not required. Module is already building for target iOS 8+

Reply:
I think we can deprecate it, certainly. But to avoid breaking changes we should just return `YES`